src/themes/teams/components/Icon/svg/icons/call.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import { IconSpec } from '../types'
+
+export default {
+  icon: classes => (
+    <svg viewBox="0 0 32 32" role="presentation" focusable="false" className={classes.svg}>
```

Comment:
Should we pass the classes to the svg element here?

Reply:
yes - but this is intentionally left to icons' maintainers, as current system (where both CSS class selectors and DOM selectors are used for styling) should be refactored